torch/csrc/jit/backends/backend_detail.h

```diff
@@ -46,6 +50,20 @@ std::function<void(Stack&)> getExecuteFunc() {
     push(stack, res);
   };
 }
+
+typedef std::function<
+    c10::IValue(const Module&, const c10::Dict<IValue, IValue>&)>
+    BackendPreprocessFunction;
+
+TORCH_API void registerBackendPreprocessFunction(
+    const std::string& name,
+    const BackendPreprocessFunction& preprocess);
+
+TORCH_API bool hasBackendPreprocessFunction(const std::string& name);
+
+TORCH_API BackendPreprocessFunction
```

Comment:
Do these need to be declared with `TORCH_API`?

Reply:
That's right.
Just registerBackendPreprocessFunction
I removed it from the others.

Reply:
Afterall, the 3 functions need to be exported with TORCH_API since:
* backend.h uses registerBackendPreprocessFunction() and backend_init.cpp uses hasBackendPreprocessFunction() and getBackendPreprocessFunction(), AND
* backend_detail.h/.cpp is in a separate .so (libtorch) from backend.h/backend_init.h/.cpp (libtorch_python)

Reply:
Thanks for following up.

For my own knowledge, what does the macro do, exactly?